SuiteCareers/SuiteCareers/obj/SuiteCareers.csproj.nuget.dgspec.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
This entire `obj` folder should be deleted from the repository, along with the  `SuiteCareers.db-shm` and `SuiteCareers.db-wal` files. You also may need to clear your cache of them, e.g.,  `git rm --cached SuiteCareers.db-shm`